### PR TITLE
mp-b2c catalog search suggestions

### PR DESCRIPTION
**Pull request by gorav-sharma-metacube**
## PR Description
CC-16148: mp-b2c catalog search suggestions

### CODE REVIEW

File: resources/environments/environments_api_mp_b2c.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
     "env": {
 
-        "glue_url": "http://glue.de.spryker.local",
+        "glue_url": "https://glue.de.mp-b2c.internal-testing.demo-spryker.com",
```
Comment (OKrivtsova) [inline resources/environments/environments_api_mp_b2c.json:6]: please revert this change

File: tests/api/mp_b2c/glue/search_endpoints/catalog_search_suggestions/positive.robot [hunk 2]
```diff
@@ -0,0 +1,262 @@
+*** Settings ***
+Suite Setup    SuiteSetup
+Test Setup    TestSetup
+Resource    ../../../../../../resources/common/common_api.robot
+Default Tags    glue
+
+*** Test Cases ***
+ENABLER
+    TestSetup
+#ET requests
```
Comment (OKrivtsova) [inline tests/api/mp_b2c/glue/search_endpoints/catalog_search_suggestions/positive.robot:10]: this comment is unneeded

File: resources/environments/environments_api_mp_b2c.json (hunk 1 above)
Comment (gorav-sharma-metacube) [reply resources/environments/environments_api_mp_b2c.json]: fixed

File: tests/api/mp_b2c/glue/search_endpoints/catalog_search_suggestions/positive.robot (hunk 2 above)
Comment (gorav-sharma-metacube) [reply tests/api/mp_b2c/glue/search_endpoints/catalog_search_suggestions/positive.robot]: fixed

File: resources/environments/environments_api_mp_b2c.json (hunk 1 above)
Comment (gorav-sharma-metacube) [reply resources/environments/environments_api_mp_b2c.json]: fixed

**Review by OKrivtsova — APPROVED** [review]
